<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 2: Окно настроек</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="table">
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>x1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dx1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dy1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>x2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dx2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dy2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Примечания:</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Загрузить из файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Выгрузить в файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Сброс</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBox_buttons"/>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="previousButton">
        <property name="text">
         <string>Шаг назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>Шаг вперед</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
